frontend/src/app/components/janus/janusVideoRoom.service.ts

```diff
@@ -234,13 +234,9 @@ export class JanusVideoRoomService {
         return this.stopAndStartSdiForwarding(shidurState, user, forwardIp, videoPorts[index], audioPort, changeTitle);
       });
 
-      this.$q.all(forwardPromises).then(() => {
-        this.$log.info('VideoRoom - all remote feeds forwarded successfully');
+      (<any> this.$q).allSettled(forwardPromises).then(() => {
```

Comment:
We report specific forwarding errors to Rollbar. I guess it "rolls back" already since if a stop or forwarding error occurres it doesnt update shidur state to reflect the corresponding state change